.github/workflows/build-and-push.yml

```diff
@@ -16,27 +16,31 @@ jobs:
       matrix:
         include:
           - dockerfile: "2.4/Dockerfile.fedora"
+            docker_context: "2.4"
             registry_namespace: "fedora"
             tag: "2.4"
             quayio_username: "QUAY_IMAGE_FEDORA_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
             image_name: "httpd-24"
 
           - dockerfile: "2.4-micro/Dockerfile.c9s"
+            docker_context: "2.4-micro"
             registry_namespace: "sclorg"
             tag: "c9s"
             image_name: "httpd-24-micro-c9s"
             quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
 
           - dockerfile: "2.4/Dockerfile.c9s"
+            docker_context: "2.4"
             registry_namespace: "sclorg"
             tag: "c9s"
             image_name: "httpd-24-c9s"
             quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
 
           - dockerfile: "2.4-micro/Dockerfile.fedora"
+            docker_context: "2.4-micro"
```

Comment:
Yeah, like here: https://github.com/sclorg/s2i-nodejs-container/blob/master/.github/workflows/build-and-push.yml#L26. Otherwise I will not have a path to README.md file. See PR here: https://github.com/sclorg/s2i-nodejs-container/pull/428